2024-BullfrogRobot/src/main/include/subsystems/Compressor.hpp

```diff
@@ -0,0 +1,18 @@
+#ifndef AIR_H
+#define AIR_H
+
+#include <frc2/command/SubsystemBase.h>
+#include <frc/PneumaticHub.h>
+#include "../Constants.h"
+
+
+class Compressor : frc2::SubsystemBase
+{
+    public:
+        void toggle_compressor();
+    private:
+        bool toggle = false;
+        };
```

Comment:
I dont think that there should be a semicolon here. Furthermore, I cant change this one, single thing, so it will have to be fixed after-the-fact.